src/pages/home/Home.js

```diff
@@ -85,6 +84,7 @@ class Home extends Component {
                     target="_blank"
                     href={`mailto:${config.contact.EMAIL}`}
                     className={styles.email}
+                    rel="noopener noreferrer"
```

Comment:
Hmm not sure why my comment didnt appear. Are we using `eslint`? If yes, there is a rule for this. Is it possible to add it?

Reply:
Currently is just a warning in dev I believe, but Travis was throwing an error that's why I added xD

Reply:
Should we it an error rather than a warning? `Tslint` also throws an error 😄

Reply:
Agreed

Reply:
Done!